tests/unit/lms/services/vitalsource/client_test.py

```diff
@@ -90,18 +90,24 @@ def test_book_info_not_found(self, svc):
         with mock.patch.object(
             VitalSourceService,
             "get",
-            side_effect=HTTPError(factories.requests.Response(status_code=404)),
+            side_effect=ExternalRequestError(
+                response=factories.requests.Response(status_code=404)
+            ),
         ):
-            with pytest.raises(ExternalRequestError):
+            with pytest.raises(ExternalRequestError) as exc_info:
                 svc.book_info("BOOK_ID")
 
+            assert exc_info.value.message == "Book BOOK_ID not found"
```

Comment:
This test wasn't actually testing that the right path was taken through the code. As a result I actually broke the code and this test was still passing. Only some missing coverage alerted me to this problem. So I added this extra assert to fix the test.